lib/flutter_crashlytics.dart

```diff
@@ -66,4 +67,52 @@ class FlutterCrashlytics {
     return await _channel.invokeMethod(
         'setUserInfo', {"id": identifier, "email": email, "name": name});
   }
+
+  /// Reports an Exception to Craslytics together with the stacktrace.
+  /// Both fields are mandatory.
+  ///
+  /// As Errors should not be cougth those are not supported.
+  ///
+  /// ```dart
+  /// try {
+  ///     // Code throwing en exception
+  /// } on Exception catch (e, s) {
+  ///     FlutterCrashlytics().logException(e, s);
+  /// }
+  /// ```
+  Future<void> logException(Exception exception, StackTrace stack) async {
+    assert(exception != null);
+    assert(stack != null);
+
+    final data = {
+      'message': exception.toString(),
+      'trace': _traces(stack),
+    };
+
+    return await _channel.invokeMethod('logException', data);
+  }
+
+  List<Map<String, dynamic>> _traces(StackTrace stack) =>
+      Trace
+          .from(stack)
+          .frames
+          .map(_toTrace)
+          .toList(growable: false);
+
+  Map<String, dynamic> _toTrace(Frame frame) {
+    final List<String> tokens = frame.member.split('.');
+
+    return {
+      'library': frame.library,
+      'line': frame.line,
```

Comment:
If this can be null like you mention on your last PR, maybe it worth adding `?? 0` ?

Reply:
The column can be `null` but apparently the line cannot be `null`

Reply:
Ok 👍